pilot/pkg/bootstrap/server.go

```diff
@@ -260,35 +269,10 @@ func (s *Server) initMonitor(args *PilotArgs) error {
 	return nil
 }
 
-func (s *Server) initClusterRegistries(args *PilotArgs) (err error) {
-	s.clusterStore = clusterregistry.NewClustersStore()
-
-	if s.kubeClient == nil {
-		log.Infof("skipping cluster registries, no kube-client created")
-		return nil
-	}
-
-	// Drop from multicluster test cases if Mock Registry is used
-	if checkForMock(args.Service.Registries) {
-		return nil
-	}
-	if args.Config.ClusterRegistriesConfigmap != "" {
-		if err = clusterregistry.ReadClusters(s.kubeClient,
-			args.Config.ClusterRegistriesConfigmap,
-			args.Config.ClusterRegistriesNamespace,
-			s.clusterStore); err != nil {
-			return err
-		}
-	}
-	log.Infof("clusters configuration %s", spew.Sdump(s.clusterStore))
-
-	return err
-}
-
-// Check if Mock's registry exists in PilotArgs's Registries
-func checkForMock(registries []string) bool {
+// Check if Kubernetes registry exists in PilotArgs's Registries
+func checkForKubernetes(registries []string) bool {
```

Comment:
why not use `hasKubeRegistry`? No need to rewrite one function.

Reply:
Ok will fix this up